server/src/core/server/graph/schema/schema.graphql

```diff
@@ -6463,6 +6499,11 @@ input MediaConfigurationInput {
   """
   twitter: TwitterMediaConfigurationInput
 
+  """
+  bluesky is the configuration for Twitter support.
```

Comment:
I think this comment should say it's for bluesky support?

Reply:
fixed